testapp/tests/conftest.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+import pytest
+from sqlalchemy import create_engine
+from testapp.models import DBSession, Base
+
+
+TEST_DATABASE_URL = 'sqlite:////tmp/test_db.sqlite'
```

Comment:
So what you're doing here is testing on a different db flavor than you are using to actually host your app. Not ideal. I would like you guys to change this to a psql db, and fix the resulting issues.